include/iterlib/LimitIterator.h

```diff
@@ -0,0 +1,31 @@
+//  Copyright (c) 2016, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
+#pragma once
+
+#include "iterlib/WrappedIterator.h"
+
+namespace iterlib {
+
+class LimitIterator : public WrappedIterator {
```

Comment:
Should we add block comments for the classes to explain the behavior of the specific iterators?